diff --git a/build/make/Makefile b/build/make/Makefile
index 92113cc..cb9d4f4 100644
--- a/build/make/Makefile
+++ b/build/make/Makefile
@@ -162,9 +162,8 @@ ifeq ($(HAVE_GNU_STRIP),yes)
 # keep them.
 %.a: %_g.a
 	$(if $(quiet),@echo "    [STRIP] $@ < $<")
-	$(qexec)$(STRIP) --strip-unneeded \
-         `$(NM) $< | grep ' [A-TV-Z] ' | awk '{print "-K"$$3'}`\
-          -o $@ $<
+	$(qexec)cp $< $@
+	$(qexec)$(STRIP) --strip-unneeded $@
 else
 %.a: %_g.a
 	$(if $(quiet),@echo "    [CP] $@ < $<")
